KinConnections/templates/connector.html

```diff
@@ -112,11 +134,12 @@ <h4 style="font-weight: 600;">Connect with {{connector['first_name']}} {{connect
               </div>
 
               <div class="my-3">
-                <div id="loadingBlock" style="display: none;" class="loading">Loading</div>
-                <div id="sentMessage" style="display: none;" class="sent-message">Your message has been sent. Thank you!</div>
+                <div class="loading">Loading</div>
+                <div class="error-message"></div>
+                <div class="sent-message">Your message has been sent. Thank you!</div>
```

Comment:
Nice touch!

Reply:
Is this working? I am not seeing code that would toggle the loader.

Reply:
do you have the loadingBlocker() form action?

Reply:
This is working. I just overlooked some stuff.